docs/porting-guides/6.x.x.md

```diff
@@ -57,14 +57,15 @@ The following data model keys have been removed from `eos_designs` in AVD 6.0.0.
 | `aaa_settings.accounting.commands[].console[].logging` | `aaa_settings.accounting.commands[].console[].methods[].method` |
 | `aaa_settings.accounting.commands[].default[].group` | `aaa_settings.accounting.commands[].default[].methods[].group` |
 | `aaa_settings.accounting.commands[].default[].logging` | `aaa_settings.accounting.commands[].default[].methods[].method` |
-| `underlay_ospf_authentication.message_digest_keys[].key` | `underlay_ospf_authentication.message_digest_keys[].cleartext_key` |
 | `avd_6_behaviors` | - |
+| `design` | - |
```

Comment:
This one probably deserves a longer entry in the porting guide - or at least pointing to the deprecation text here: https://avd.arista.com/5.7/docs/porting-guides/5.x.x.html#deprecation-of-designtype 

Also we need to adjust the `input_variables.md`  as right now it reads:

<img width="1684" height="413" alt="Screenshot 2025-09-18 at 09 50 52" src="https://github.com/user-attachments/assets/1263c5c3-0d86-418f-9bd1-c1585cbafa37" />

@ClausHolbechArista